site/e2e/tests/users/createUserWithPassword.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { test, expect } from "@playwright/test";
```

Comment:
I don't know what the convention is, but if you want to merge both test files into a single `e2e/test/users.spec.ts`, I'm cool with that.

Reply:
Sounds reasonable.